packages/cli-kit/src/analytics.ts

```diff
@@ -55,7 +59,7 @@ const buildHeaders = (currentTime: number) => {
   }
 }
 
-const buildPayload = async (command: string, args: string[] = [], currentTime: number) => {
+const buildPayload = async (errorMessage: string | undefined, currentTime: number) => {
```

Comment:
What about using an options type here to make it more obvious from the caller what arguments are? Looking at the following code it's hard to tell that the first string is an error message:

```js
analytics.reportEvent("foo")
```